homeassistant/components/binary_sensor/wink.py

```diff
@@ -36,54 +36,65 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     import pywink
 
     for sensor in pywink.get_sensors():
-        _id = sensor.object_id() + sensor.name()
+        _id = sensor.object_id() + "_" + sensor.capability()
```

Comment:
Use `"{}_{}".format(sensor.object_id(), sensor.capability())` for bulding strings